microcosm_pubsub/dispatcher.py

```diff
@@ -48,7 +48,7 @@ def handle_message(self, message):
         Handle a single message.
 
         """
-        media_type = message["media_type"]
+        media_type = message.get("media_type", "_")
```

Comment:
It probably makes more sense for `SQSMessage.from_sqs` to return a `(media_type, message)` tuple and have both args get passed here. It would be the responsibility of the "envelope decoding" function to extract this media type; in this case, you'd inject a function that returned: `"_", loads(message)`